README.md

```diff
@@ -395,6 +395,13 @@ not the class should stay or go on the vnode.
 h("a", { class: { active: true, selected: false } }, "Toggle");
 ```
 
+In JSX, you can use the `class` prop.
```

Comment:
I think "attribute" is more appropriate than "prop" in a JSX context?

Reply:
I've been thinking about this since you commented yesterday and I'm not sure! From data model perspective, the "key" (for lack of a better term) is neither a property nor an attribute, but can produce either of those as side-effects.

My original verbiage was "key"; does that make more sense? Or should it be labeled in relation to its side-effect? E.g., "attribute" for `class` and `style`, "prop" for `dataset`, and "handler" for `on`. 

Thanks for the review!

Reply:
I was thinking "attribute" because in JSX something like `<div foo=bar>` looks like HTML/XML and in those languages "foo" would be called an attribute. But, I see your point, those "attributes" may not actually correspond to the "real" "attributes" on the actual HTML/DOM.

In the [JSX specification](https://facebook.github.io/jsx/) it seems that these things are called attributes, regardless of what they eventually end up meaning to whoever is using JSX.

But this doesn't really matter, I think people will get the point anyway 😅. What about sidestepping the issue an writing something like

> In JSX, you can use `class` like this:

?. Alternatively, I think "key" would also be fine. But the abbreviated "prop" sounds weird to me.

Reply:
Updated using your suggested verbiage:

> In JSX, you can use <module> like this:

I also added to the event listeners section, forgot to do that before.

Reply:
My commit message wasn’t helpful on that last commit, but I suppose you will probably squash commits when merging for simplicity. :)

Reply:
Looks good. Thanks for the PR 😀